src/lib.rs

```diff
@@ -58,8 +58,13 @@
 // Clippy does not understand that generic numeric types are not always
 // as capable as built-in ones and raise false warnings
 #![cfg_attr(feature = "cargo-clippy", allow(clippy::op_ref))]
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
```suggestion
#![no_std]
#[cfg(feature = "std")]
extern crate std;
```

This will make both `std` and `core` visible in all modules and remove the need for `use` gating by `feature = "std"`.